pkg/csi/service/vanilla/controller.go

```diff
@@ -1324,120 +1324,192 @@ func (c *controller) createBlockVolumeWithPlacementEngineForMultiVC(ctx context.
 		topologyRequirement *csi.TopologyRequirement
 		combinedErrMssgs    []string
 	)
-	// Get accessibility requirements.
-	topologyRequirement = req.GetAccessibilityRequirements()
-	if topologyRequirement == nil {
-		return nil, csifault.CSIInvalidArgumentFault, logger.LogNewErrorCode(log, codes.InvalidArgument,
-			"accessibility requirements cannot be nil for a multi-VC environment")
-	}
+
 	var multivCenterTopologyDeployment bool
 	if len(c.managers.VcenterConfigs) > 1 {
 		multivCenterTopologyDeployment = true
 	}
 	vcTopologySegmentsMap := make(map[string][]map[string]string)
 	if multivCenterTopologyDeployment {
+		// Get accessibility requirements.
+		topologyRequirement = req.GetAccessibilityRequirements()
```

Comment:
can we initialize topologyRequirement outside if check and use it wherever applicable within the function, instead of req.GetAccessibilityRequirements()?

Reply:
done